libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/InfoCallout/InfoCallout.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Callout, IconButton } from "office-ui-fabric-react";
+import React from "react";
+
+import { infoCalloutStyles } from "./InfoCallout.styles";
+
+export interface IInfoCalloutProps {
+  iconId: string;
+  infoText: string;
+  title: string;
+}
+
+export interface IInfoCalloutState {
+  isCalloutVisible: boolean;
+}
+
+export class InfoCallout extends React.Component<
+  IInfoCalloutProps,
+  IInfoCalloutState
+> {
+  public constructor(props: IInfoCalloutProps) {
+    super(props);
+    this.state = {
+      isCalloutVisible: false
+    };
+  }
+  public render(): React.ReactNode {
+    const classNames = infoCalloutStyles();
+    return (
+      <span>
+        <IconButton
+          id={this.props.iconId}
+          iconProps={{ iconName: "Info" }}
+          title={this.props.title}
+          onClick={this.onIconClick}
+          styles={{
+            root: {
+              color: "rgb(0, 120, 212)",
+              height: "24px",
+              marginBottom: -3,
+              marginTop: -3
+            }
+          }}
+        />
+        {this.state.isCalloutVisible && (
+          <Callout
+            target={"#" + this.props.iconId}
+            setInitialFocus={true}
+            onDismiss={this.onDismiss}
+            role="alertdialog"
+          >
+            <div className={classNames.calloutInfo}>
+              <div>
```

Comment:
why two `div`s? The inner one doesn't seem to add anything. What am I missing?

Reply:
Oh, this was used when I had a button in it, but I didn't remove the extra div when I removed the button.  Removed it now, thanks!